### PR TITLE
[NEW] Show version name/code in about screen

### LINKED ISSUE #920: [IMPROVEMENT] Show version name/number under settings

### PR DESCRIPTION
**Pull request by divyanshub024**
@RocketChat/android

Closes #920 #962 

about-eng

### CODE REVIEW

**Comment by divyanshub024** [comment]
@rafaelks can you try to run CI on this PR.

**Review by rafaelks — APPROVED** [review]
Screen looks good to me. Can you review this PR please @leonardoaramaki?

File: app/build.gradle [hunk 1]
```diff
@@ -65,6 +65,7 @@ dependencies {
 
     implementation libraries.dagger
     implementation libraries.daggerSupport
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```
Comment (Shailesh351) [inline app/build.gradle]: It is already added as dependency at https://github.com/divyanshub024/Rocket.Chat.Android/blob/530ff7c7439822851dcdeaaa5df14bc7e475bb08/app/build.gradle#L60

Comment (Shailesh351) [reply app/build.gradle]: Everything else looks good to me :+1:

Comment (divyanshub024) [reply app/build.gradle]: Thanks, @Shailesh351

**Comment by divyanshub024** [comment]
@leonardoaramaki This PR is ready for review.

**Review by philipbrito — COMMENTED** [review]
I don't like the idea of having that assets on our Android app when we can use Vector Drawables instead. That sources increases our app size a lot. For the app logo here, you can see the [ic_launcher_foreground.xml](https://github.com/RocketChat/Rocket.Chat.Android/blob/develop-2.x/app/src/main/res/drawable/ic_launcher_foreground.xml) already in the app as an example.

**Comment by divyanshub024** [comment]
@filipedelimabrito I have made the changes

**Comment by philipbrito** [comment]
Thanks for this PR @divyanshub024!

**Pull request merged**